src/actions/account.js

```diff
@@ -325,7 +325,16 @@ export const { addAccessKey, createAccountWithSeedPhrase, addAccessKeySeedPhrase
     CREATE_ACCOUNT_WITH_SEED_PHRASE: [
         async (accountId, recoveryKeyPair, fundingContract, fundingKey) => {
             await wallet.saveAccount(accountId, recoveryKeyPair);
-            await wallet.createNewAccount(accountId, fundingContract, fundingKey, recoveryKeyPair.publicKey)
+
+            try {
+                await wallet.createNewAccount(accountId, fundingContract, fundingKey, recoveryKeyPair.publicKey)
+            } catch (e) {
+                console.warn('RPC error: createNewAccount\n', e)
+                await new Promise((r) => setTimeout(r, 4000)) // wait 4s for chain
```

Comment:
If account creation failed with RetriesExceeded – it generally shouldn't help to wait 4 more seconds (as `RetriesExceeded` should take far longer that that). If default timeout values in near-api-js are too small – we can just increase them.